config/settings.yml

```diff
@@ -78,6 +78,9 @@
   :use_vim_broker: true
 :container_scanning:
   :concurrent_per_ems: 3
+  :no_proxy: "localhost,127.0.0.1"
+  :http_proxy: "http://proxy:80"
+  :https_proxy: "http://proxy:80"
```

Comment:
Are these examples for the user to change? won't they break scanning since they use a proxy that does not exist?